weeather/LoadBots.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text.Json;
+using weather.Entities;
+
+namespace weather
+{
+    internal class LoadBots
+    {
+
+        public List<Bot> LoadBotsFromConfigFile()
+        {
+            string data = ConfigurationFileData();
+            return ParseToBots(data);
+        }
+
+        private string ConfigurationFileData()
+        {
+            string fileData = File.ReadAllText("../../../config.json");
+            return fileData;
+        }
+
+        //here i had to do it like this since i will not be able to pass the bot in 
+        //bitConfiguration to a new method which caused me to preform all the work in a single method
+        private List<Bot> ParseToBots(string data)
+        {
+            List<Bot> bots = new List<Bot>();
+            var botConfigurations = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(data);
```

Comment:
I can see that the logic here handles json only, the requirements of the task states that configs can either be json or xml, can you please handle this ?

Reply:
Aren't the different file formats for weather data? It says that the configuration file will be in JSON format 
quoting  "The configuration file should be in a JSON format"